### ISSUE TITLE
Wallet balance

### ISSUE THREAD

**Issue by buckyroberts** [buckyroberts/Vataxia-Frontend#52]
Bug getting wallet balance since primary key was changed

**Comment by PaulsBecks** [comment]
Yes, I think this bug is due to the fact that ``` wallet.id != user.id```

This would be a thing to change in the back-end, I guess. To get around this problem one could ad the following in ```Vataxia-Frontend/src/actions/credits/wallet/get.js```

after 
``` js
  const response = await axios.get(`${settings.API_ROOT}/wallets/${userId}`, tokenHeader());
```
ad
``` js
  response.data['_id'] = response.data['id']
  response.data['id'] = response.data['user']['id']
```

Its now already a year ago, but might be still useful to someone.
